app/service/data_svc.py

```diff
@@ -65,6 +65,16 @@ def __init__(self, dao):
                 for fact in source['facts']:
                     fact['source_id'] = source_id
                     await self.create_fact(**fact)
+                relationships = source.get('relationships')
+                if relationships:
+                    for _, relationship in relationships.items():
```

Comment:
you can just iterate through values for x in dict.values()

for value in a_dict.values():
    print(value)